### PR TITLE
ActionsCell added to table and resultsetTable

### PR DESCRIPTION
**Pull request by Jialecl**
**Checklist**
_(Check off all the items before submitting)_

- [x] Build process is done without errors. All tests pass in the `/lib` directory.
- [x] Self-reviewed the code before submitting.
- [x] Meets accessibility standards.
- [ ] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**
Action Icon added to the table and resultsetTable to be used as a compound component.

**Description**
The component will be composed by the whole cell.
The action icon can also be a trigger for a dropdown, so the options prop was added to the compound component.

### CODE REVIEW

File: lib/src/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -242,10 +242,7 @@ const calculateWidth = (margin, size) =>
     : sizes[size];
 
 const DropdownContainer = styled.div<{ margin: DropdownPropsType["margin"]; size: DropdownPropsType["size"] }>`
-  display: inline-block;
   width: ${(props) => calculateWidth(props.margin, props.size)};
-  text-overflow: ellipsis;
-  overflow: hidden;
```
Comment (Mil4n0r) [inline lib/src/dropdown/Dropdown.tsx:248]: This removes a previously existing margin. Has it been validated by the design team?

File: lib/src/common/variables.ts [hunk 2]
```diff
@@ -284,8 +284,14 @@ export const componentTokens = {
...
     buttonPaddingLeft: "16px",
     buttonPaddingRight: "16px",
+    buttonHeight: "40px",
+    buttonBorderRadius: "4px",
+    buttonBorderStyle: CoreTokens.border_none,
+    buttonBorderThickness: CoreTokens.border_width_0,
+    buttonBorderColor: CoreTokens.color_transparent,
     disabledColor: CoreTokens.color_grey_500,
     disabledButtonBackgroundColor: CoreTokens.color_transparent,
+    disabledButtonBorderColor: CoreTokens.color_transparent,
```
Comment (Mil4n0r) [inline lib/src/common/variables.ts:294]: Adding new Component Tokens may require updates in advanced-theme / advanced-schema

File: lib/src/common/variables.ts [hunk 3]
```diff
@@ -867,6 +873,16 @@ export const componentTokens = {
...
+    actionIconColor: CoreTokens.color_purple_700,
+    disabledActionIconColor: CoreTokens.color_grey_500,
+    hoverActionIconColor: CoreTokens.color_purple_700,
+    focusActionIconColor: CoreTokens.color_purple_700,
+    activeActionIconColor: CoreTokens.color_purple_700,
+    actionBackgroundColor: CoreTokens.color_transparent,
+    disabledActionBackgroundColor: CoreTokens.color_transparent,
+    hoverActionBackgroundColor: CoreTokens.color_grey_100,
+    focusActionBorderColor: CoreTokens.color_blue_600,
+    activeActionBackgroundColor: CoreTokens.color_grey_300,
```
Comment (Mil4n0r) [inline lib/src/common/variables.ts:885]: Adding new Component Tokens may require updates in advanced-theme / advanced-schema

**Review by Mil4n0r — CHANGES_REQUESTED** [review]

File: lib/src/common/variables.ts (hunk 2 above)
Comment (Jialecl) [reply lib/src/common/variables.ts:294]: Those are in website, unless these changes are merged we can not change the ones in website

File: lib/src/common/variables.ts (hunk 3 above)
Comment (Jialecl) [reply lib/src/common/variables.ts:885]: Yes that should be added in a different PR

**Pull request merged**